frontend/src/components/list-card/task-list/task-list.js

```diff
@@ -2,27 +2,54 @@ import styles from "./task-list.module.css";
 import CloseButton from "../../close-button/close-button";
 import useActiveList from "../../../hooks/use-active-list";
 
+import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd';
+
 const TaskList = () => {
-    const {removeTask, tasks, uncheckTask, completeTask} = useActiveList()
+    let {removeTask, tasks, uncheckTask, completeTask, updateTasksOrder} = useActiveList();

```

Comment:
use const instead of let

Reply:
Finally I need let instead of const beacuse I need to asign new ordered array to tasks variable